string request
---
spectrometer_drivers/Spectra response